src/main/java/org/folio/inventory/storage/external/MultipleRecordsFetchClient.java

```diff
@@ -0,0 +1,98 @@
+package org.folio.inventory.storage.external;
+
+import static org.apache.commons.collections4.ListUtils.partition;
+import static org.folio.inventory.support.JsonArrayHelper.toList;
+import static org.folio.util.StringUtil.urlEncode;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.folio.inventory.storage.external.exceptions.ExternalResourceFetchException;
+import org.folio.inventory.support.http.client.Response;
+
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+
+public class MultipleRecordsFetchClient {
+  private static final int DEFAULT_PARTITION_SIZE = 30;
```

Comment:
Why does DEFAULT_PARTITION_SIZE equal 30?

Reply:
The jira says that the failure occurred when there are more than 50 instances, so I decided to chose a middle value.

Do you think it should be changed?